tests/test_gridsearch.py

```diff
@@ -100,7 +100,7 @@ def test_grid_search(self):
 
         # Test exception handling on scoring
         grid_search.scoring = "sklearn"
-        with self.assertRaises(ValueError):
+        with self.assertRaises((KeyError, ValueError)):
```

Comment:
Where does the `KeyError` come from?

Reply:
Should be because of the _check_multimetric_scoring. I think this might've always happened though.

Reply:
I would be wary of just changing the tests because I feel like the KeyError would have been mentioned before since sklearn supports multimetric scoring too

Reply:
We could just change it back to ValueError and I think it would be fine

Reply:
Ok, let's try that and see how it goes